tools/probe/probe_input_names_strict.py

```
@@ -0,0 +1,177 @@
+
+            try:
+                raw_names = list(temp_node.inputNames())
+
+                if len(raw_names) == 0:
+                    stats["no_names_returned"] += 1
+                else:
+                    stats["has_valid_names"] += 1
+                    if len(samples[cat_name]) < 3:
+                        samples[cat_name].append((node_type_name, raw_names))
+
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)